tests/lib/pkgdb.sh

```diff
@@ -279,13 +279,13 @@ distro_install_build_snapd(){
         packages=
         case "$SPREAD_SYSTEM" in
             ubuntu-*|debian-*)
-                packages="${GOHOME}"/snapd_*.deb
+                packages="${GOHOME}/snapd_*.deb"
```

Comment:
Hmm, does it really do the right thing? Compare `ls "$HOME/*"` and `ls "$HOME"/*`?